sbomgrader/core/utils.py

```diff
@@ -46,15 +50,86 @@ def get_path_to_var_transformers(schema_path: str | Path) -> Path:
     return schema_path.parent / "transformers" / schema_path.name.split(".", 1)[0]
 
 
+def get_path_to_preprocessing(schema_path: str | Path) -> Path:
+    if isinstance(schema_path, str):
+        schema_path = Path(schema_path)
+    return schema_path.parent / "preprocessing" / schema_path.name.split(".", 1)[0]
+
+
+def get_path_to_postprocessing(schema_path: str | Path) -> Path:
+    if isinstance(schema_path, str):
+        schema_path = Path(schema_path)
+    return schema_path.parent / "postprocessing" / schema_path.name.split(".", 1)[0]
+
+
+def get_path_to_module(
+    schema_path: str | Path,
+    kind: Literal["transformer", "preprocessing", "postprocessing"],
+    first_or_second: Literal["first", "second"],
+    sbom_format: "sbomgrader.core.formats.SBOMFormat,",
+):
+    map_ = {
+        "transformer": get_path_to_var_transformers,
+        "preprocessing": get_path_to_preprocessing,
+        "postprocessing": get_path_to_postprocessing,
+    }
+    mod_func = map_.get(kind)
+    if not mod_func:
+        raise ValueError(f"Wrong kind value: {kind}")
+    mod_dir = mod_func(schema_path)
+    file = None
+    for filename in (
+        f"{first_or_second}.py",
+        f"{sbom_format.value}.py",
+    ):
+        f = mod_dir / filename
```

Comment:
I know that this is your code style but I think you should avoid having a one letter variables names. You should always give it a more reasonable name. Tha same applies to a variable `map_`. For a reviewer it is always easier to see imidiatelly what the map is used for. So having a context in the variable name is always helpful.